policy/scan/local_scanner.go

```diff
@@ -391,6 +398,10 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 					return nil, false, multierr.Wrap(err, "failed to generate a random asset MRN")
 				}
 				cur.asset.Mrn = x.String()
+				assetName := viper.GetString("asset-name")
```

Comment:
same comment as above

Reply:
actually, i don't think this is even needed. the `assets` list gets modified. above as well and the name gets assigned